galleries/examples/tag_guidelines.rst

```diff
@@ -0,0 +1,67 @@
+**Guidelines for assigning tags to gallery examples**
+
+*Why do we need tags?*
+
+    Tags serve multiple purposes.
+
+    Tags have a one-to-many organization (i.e. one example can have several tags), while the gallery structure requires that examples are placed in only one location. This means tags provide a secondary layer of organization and make the gallery of examples more flexible and more user-friendly.
+
+    They allow for better discoverability, search, and browse functions. They are helpful for users struggling to write a search query for what they're looking for.
+
+    Hidden tags provide additional functionality for maintainers and contributors.
+
+*What gets a tag?*
+
+    Every gallery example should be tagged with:
+
+    * 1+ content tags
+    * structural, domain, or internal tag(s) if helpful
+
+        Examples with many tags may indicate that the entry is a showcase example.
+
+        Tags can repeat existing forms of organization (e.g. an example is in the Animation folder and also gets an ``animation`` tag).
+
+        Tags are helpful to denote particularly good "byproduct" examples. E.g. the explicit purpose of a gallery example might be to demonstrate a colormap, but it's also a good demonstration of a legend. Tag ``legend`` to indicate that, rather than changing the title or the scope of the example.
```

Comment:
```suggestion
Every gallery example should be tagged with:

* 1+ content tags
* structural, domain, or internal tag(s) if helpful

Examples with many tags may indicate that the entry is a showcase example.

Tags can repeat existing forms of organization (e.g. an example is in the Animation folder and also gets an ``animation`` tag).

Tags are helpful to denote particularly good "byproduct" examples. E.g. the explicit purpose of a gallery example might be to demonstrate a colormap, but it's also a good demonstration of a legend. Tag ``legend`` to indicate that, rather than changing the title or the scope of the example.
```